mentat/parsers/git_parser.py

```diff
@@ -62,6 +100,12 @@ def parse_string(self, git_diff: str) -> ParsedLLMResponse:
                 is_deletion=is_deletion,
                 rename_file_path=new_name,
             )
+            if not is_creation:
+                file_edit.previous_file_lines = (
```

Comment:
This is used specifically for undo's (when the fileedit is applied, it sets this so that the fileedit can be undone). It feels a bit weird using it here, but I guess it works.